tests/integration/java-integration/src/test/java/io/fluentbit/test/MyLogger.java

```diff
@@ -0,0 +1,68 @@
+/*  Fluent Bit
+ *  ==========
+ *  Copyright (C) 2019-2021 The Fluent Bit Authors
+ *  Copyright (C) 2015-2018 Treasure Data Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.fluentbit.test;
+
+import org.komamitsu.fluency.Fluency;
```

Comment:
What's the licensing of this?

Reply:
Apache 2.0: https://github.com/komamitsu/fluency/blob/master/LICENSE.txt

Reply:
OK good, can you add it to the PR description just so it is more visible?

Reply:
done